src/Homie/Boot/BootNormal.cpp

```diff
@@ -97,6 +97,9 @@ void BootNormal::loop() {
     ESP.restart();
   }
 
+  for (HomieNode* iNode : HomieNode::nodes) {
```

Comment:
Why is this moved? It may break functionality that assumes that loopFunction is executed before the individual node loop methods?

Reply:
After the loop is before the loop.

Prior the change, the loop-function was called as last. Now it is called first, so the order has not changed.

Reply:
OK. Maybe it's not so bad, but the end of the loop and the start are only the same if there's nothing else in the Arduino loop function.